src/brace_cleanup.cpp

```diff
@@ -691,7 +691,7 @@ static void parse_cleanup(parse_frame_t *frm, chunk_t *pc)
       LOG_FMT(LSTMT, "%s(%d): %zu> reset expr on %s\n",
               __func__, __LINE__, pc->orig_line, pc->text());
    }
-   else if (pc->type == CT_BRACE_CLOSE)
+   else if (pc->type == CT_BRACE_CLOSE && pc->pp_level == 0)
```

Comment:
Yes, that header no longer fails. What's the best way through this-- check and adapt for ifdef_over_whole_file() ?

On the other hand, that code will not compile... maybe it's easier and better to let the compiler catch such a problem and not leave as defined behavior how Uncrustify will handle such an input? If so then the whole error message could be removed.